src/server/test-helpers/mocks.js

```diff
@@ -18,7 +18,10 @@ export const mockExemption = {
   activityDescription: 'Test activity description',
   // consent:yes: means consent to withdraw the application from the public register
   publicRegister: { consent: 'yes', reason: 'Test reason' },
-  multipleSiteDetails: { multipleSitesEnabled: false },
+  multipleSiteDetails: {
+    multipleSitesEnabled: false,
+    sameActivityDates: 'yes'
```

Comment:
Is this an invalid combination? Multiple sites disabled, but same activity dates is yes